packages/playground/src/components/vm_deployment_table.vue

```diff
@@ -197,18 +203,20 @@ async function loadDeployments() {
   if (chunk2.count > 0 && migrateGateways) {
     await migrateModule(grid!.gateway);
   }
+  let chunk3: LoadedDeployments<any[]> = { count: 0, items: [], failedDeployments: [] };
+  if (showAllDeployments.value) {
```

Comment:
Shouldn't this be applied in k8s deployments as well?

Reply:
thanks applied, please test it